specs/BrowserTaskManager.md

```diff
@@ -0,0 +1,69 @@
+# Background
+The WebView2 team has been asked to provide an API to expose the Browser Task
+Manager. At the current time, it is expected that the user will close the Task 
```

Comment:
It would be good to give a little more info:
```suggestion
Manager. The Browser Task Manager is a window helpful for debugging that displays the different processes associated with the current browser process and what they're used for. For instance, that a particular process is a renderer process and all of the different web pages rendered by that process. In a chromium browser it can be opened by the end user by pressing `Shift+Esc` or from the browser's title bar's context menu's `Browser task manager` entry.

At the current time, it is expected that the user will close the Task 
```